go/dpagg/sum_test.go

```diff
@@ -1072,11 +1075,11 @@ func TestSumComputeConfidenceIntervalForFloat64Computation(t *testing.T) {
 		c.Result()
 		got, _ := c.ComputeConfidenceInterval(0.5)
 
-		if !ApproxEqual(got.LowerBound, tc.want.LowerBound) {
+		if math.Abs(got.LowerBound-tc.want.LowerBound) > tc.tolerance*math.Max(got.LowerBound, tc.want.LowerBound) {
```

Comment:
why we can't use ApproxEqual?

Reply:
Gaussian has only at most 3 significant digits accuracy thus it needs a tolerance of 1e-3 while Laplace is as accurate as it gets. Now because approxEqual doesn't have a parameter for tolerance and this is the only test which needs one, I have written it in the if condition.

Reply:
I see, I would still add new method which accepts tolerance as a parameter (might be useful in the future) and the old method "ApproxEqual" I would rewrite - so it would call our new method with default tolerance. WDYT?

Reply:
Added TODO so that these test utilities can be fixed for all packages later.